homeassistant/components/device_tracker/__init__.py

```diff
@@ -467,10 +473,9 @@ def update_config(path: str, dev_id: str, device: Device):
 
         for key, value in (('name', device.name), ('mac', device.mac),
```

Comment:
Tried it on a per property level, but it added whitespace. On a per device level it should be ok 👍